cpp/velox/memory/VeloxMemoryPool.cc

```diff
@@ -65,17 +61,17 @@ std::shared_ptr<velox::memory::MemoryUsageTracker> createMemoryUsageTracker(
 
 //  The code is originated from /velox/common/memory/Memory.h
 //  Removed memory manager.
-class WrappedVeloxMemoryPool final : public velox::memory::MemoryPool {
+class VeloxMemoryPool final : public velox::memory::MemoryPool {
```

Comment:
There is `WrappedArrowMemoryPool`, let's use rename that one as well?

Reply:
good suggestion